### DISCUSSION TITLE
Install Help

### DISCUSSION THREAD

**Discussion by SoccerFox** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#138] [Q&A]
I am new to Linux and want to install this on my PT315-53, but I don't know how to install it could you provide install instructions in the description of the Project?

**Comment by JafarAkhondali** [comment]
What is your Linux distro?  You can check it with this command `lsb_release -a`

**Reply by SoccerFox** [reply]
Ubuntu, do you need the version?

**Reply by JafarAkhondali** [reply]
Here is the installation command:
```
git clone https://github.com/JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module
cd "acer-predator-turbo-and-rgb-keyboard-linux-module"
chmod +x ./*.sh
sudo ./install_service.sh
```
It's also available in the readme
